app/src/ui/changes/changes-list.tsx

```diff
@@ -377,6 +379,31 @@ export class ChangesList extends React.Component<
     showContextualMenu(items)
   }
 
+  private getNamesOfSelectedOversizedFiles = async () => {
+    const fileNames: string[] = []
```

Comment:
:art: no need to be explicit about the signature: `const fileNames = new Array<string>()`